### ISSUE TITLE
Single layer Keras LSTM converted to loop instead of ONNX LSTM op

### ISSUE THREAD

**Issue by JonTriebenbach** [onnx/tensorflow-onnx#1851]
The attached example shows a Keras model with a single layer LSTM that is converted to a loop instead of the expected ONNX LSTM op.

Test system configuration:
Using `tensorflow/tensorflow:2.7.1` docker image:
```
tensorflow-cpu               2.7.1
tf2onnx                      1.10.0
```

This issue appears to be similar to **tf.keras.layers.LSTM not converted to ONNX LSTM layer** #1546 with pull request [Add Keras LSTM support ](https://github.com/onnx/tensorflow-onnx/pull/1752).

[onnx_issue.zip](https://github.com/onnx/tensorflow-onnx/files/8073134/onnx_issue.zip)

Please review this issue.  Thanks.

**Comment by hwangdeyu** [comment]
This PR [Add Keras LSTM support ](https://github.com/onnx/tensorflow-onnx/pull/1752) involved in `tf2onnx verison == 1.9.3`.
So This issue should be fixed after tf2onnx == 1.9.3 theoretically.
I also tried to test it in tensorflow==2.7.1 and tfonnx == 1.9.3/1.10.0, and the lstms are both LSTM op instand of loop..
![image](https://user-images.githubusercontent.com/10047193/154437563-6d52760a-a9fa-4d66-9d7b-8005f412267d.png)
Could you check it again? Thanks.

**Comment by JonTriebenbach** [comment]
@hwangdeyu Yes, we have other LSTM models that now convert as expected with tf2onnx 1.9.3.  Thank you for those fixes that corrected those other models.  The problem is that this new model is not converting as expected.  The problem shows on both 1.9.3, and on the latest 1.10.0.

I do not know what is different about this new model that is not converting as expected.  We have other models that have multiple LSTM layers, which convert as expected, this model is only a single layer LSTM model, which indicates it is not a multi-layer model issue.  There is some other complexity in this new model that is causing a conversion issue.

Please continue to investigate.  Thank you.

**Comment by mjlorenzo305** [comment]
any updates on this one?

**Comment by AndreyOrb** [comment]
Is there any progress with this issue?

**Comment by AndreyOrb** [comment]
Is there any ETA on this issue?

**Comment by fatcat-z** [comment]
> Is there any ETA on this issue?

Not yet.
